src/ApiResponseHelpers.php

```diff
@@ -33,17 +36,23 @@ public function respondNotFound(
     /**
      * @param array|Arrayable|JsonSerializable|null $contents
      */
-    public function respondWithSuccess($contents = []): JsonResponse
+    public function respondWithSuccess($contents = null): JsonResponse
     {
-        $contents = $this->morphToArray($contents);
+        $contents = $this->morphToArray($contents) ?? [];
```

Comment:
Change of default value won't impact calls without arguments, but will allow to cleanly change passed nulls to empty arrays when someone does pass null. Empty arrays will stay the same as before.